aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -2425,52 +2304,20 @@ module aptos_framework::delegation_pool {
         buy_in_pending_inactive_shares(pool, delegator_address, amount);
     }
 
-    #[test_only]
-    public fun enable_delegation_pool_allowlisting_feature(aptos_framework: &signer) {
-        features::change_feature_flags_for_testing(
-            aptos_framework,
-            vector[features::get_delegation_pool_allowlisting_feature()],
-            vector[]
-        );
-    }
-
     #[test(aptos_framework = @aptos_framework, validator = @0x123)]
-    #[expected_failure(abort_code = 0x3000A, location = Self)]
-    public entry fun test_delegation_pools_disabled(
-        aptos_framework: &signer,
-        validator: &signer,
-    ) acquires DelegationPool, GovernanceRecords, BeneficiaryForOperator, NextCommissionPercentage {
-        initialize_for_test(aptos_framework);
-        features::change_feature_flags_for_testing(aptos_framework, vector[], vector[DELEGATION_POOLS]);
-
-        initialize_delegation_pool(validator, 0, vector::empty<u8>());
-    }
-
-    #[test(aptos_framework = @aptos_framework, validator = @0x123)]
-    public entry fun test_set_operator_and_delegated_voter(
+    public entry fun test_set_operator(
```

Comment:
we should probably start not having tests be `public entry` so that, people don't end up making accidental functions that can't be removed / other issues